doc/adr/0012-node-connection-info.md

```diff
@@ -0,0 +1,18 @@
+# 11. Decline Node Request
+
+Date: 11/03/2021
+
+## Status
+
+Accepted
+
+## Context
+We want to be able to display more info regarding the status of the nodes connected to the network. A useful feature would be to display if a node is still connected or not and also to see a "last seen" timestamp of the nodes.
+
+
+## Decision
+In order for the connection status to be displayed, an endpoint checking the multichain's [getpeerinfo](https://www.multichain.com/developers/json-rpc-api/) function is called. This returns the nodes connected at the moment. However, since MultiChain doesn't provide information about when a node was last seen we had to implement this feature in the api. A new stream was created called `network_log` was created and every 24h an event is saved on the stream displaying the addresses of the nodes connected at that moment. All nodes in the network perform the check but only save it on the stream if there is no event saved for that day. This way, the check is being recorded even if one of the nodes is disconnected but it is also prevents duplicate entires.
```

Comment:
use present tense e.g. we have to implement or a new stream is/shall be created ... aso